TypeCobol.Test/Parser/Performance/Performance.cs

```diff
@@ -368,8 +368,21 @@ public void Part3_FullParsing_TC_DeepTypes()
 
 
 
-
-
+        /// <summary>
+        /// Method for parsinga document.
+        /// </summary>
+        /// <param name="fullPath"></param>
+        /// <param name="options"></param>
+        /// <param name="format"></param>
+        /// <param name="copiesFolder"></param>
+        /// <returns></returns>
+        protected virtual TypeCobol.Parser ParseDocument(string fullPath, TypeCobolOptions options, TypeCobol.Compiler.DocumentFormat format, string[] copiesFolder)
```

Comment:
This method seems only for FullParsing. Does it also include Incremental ?

Reply:
You are right, done in 3b1d62e